Content.Client/UserInterface/Systems/Guidebook/GuidebookUIController.cs

```diff
@@ -192,6 +194,14 @@ public void OpenGuidebook(
                 RecursivelyAddChildren(guide, guides);
             }
         }
+        if (guides.ContainsKey(_lastEntry))
+        {
+            selected ??= _lastEntry;
+        }
+        else
+        {
+            selected ??= _configuration.GetCVar(CCVars.DefaultGuide);
+        }
```

Comment:
I would rewrite this code to wrap the entire thing in an `if (selected == null)` and make the `??=` operators regular `=`. This would make the logic more obvious in my opinion.